### DISCUSSION TITLE
Inertia Adapter

### DISCUSSION THREAD

**Discussion by Joehoel** [47ng/nuqs#786] [Q&A] [answered]
Is there any work being done on an adapter for Inertia? If there is not, I would love to get started on that.

I saw it being mentioned in [this](https://github.com/47ng/nuqs/discussions/620#discussioncomment-10514227) comment, so I thought there would be some interest for it.

**Comment by franky47** [comment, marked as answer]
There was some discussion about this a month ago: https://x.com/nuqs47ng/status/1851170879566160091

Feel free to open a PR!
